src/main/java/com/google/sps/python/heatMap.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import json
+popFile = 'datasets/county-population.txt'
+caseFile = 'datasets/by-county-and-date.txt'
+<<<<<<< HEAD
+coordFile = 'datasets/usa_county_wise.txt'
+ignoreLocations = [96, 193, 1863]  # Edge case locations where I know the population is 0 (ex: Princess Cruise Ship)
+us_state_abbrev = {
```

Comment:
Use same capitalization as suggested above since this can be considered GLOBAL_CONSTANT (only the variable name itself, the strings are fine).